src/app/story/storysidebar/story.sidebar.component.spec.ts

```diff
@@ -8,6 +8,7 @@ import { PropertyDisplayPipe, PropertyValueDisplayPipe } from '../../pipes/prope
 import { bug } from '../test.data';
 import { TooltipModule } from 'ngx-bootstrap/tooltip';
 import { RouterTestingModule } from '@angular/router/testing';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```

Comment:
Non-local imports should be in the first group of imports. It seems like this rule was broken previous to your PR though.